recipes/functional_algorithms/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "functional_algorithms" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/functional_algorithms-{{ version }}.tar.gz
+  sha256: 5ab6b44cd72e2d0bdace4a01ee8013f7a8cc1e84d3145edb9bff46997e4c5ce0
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.11
+    - setuptools >=64
+    - setuptools-scm >=8
+    - pip
+  run:
+    - python >=3.11
+    - black
```

Comment:
Is this really required? Black is normally not a typical runtime dependency.

Reply:
Yes, the package generates Python code and `black.format_str` is used to format the results.